pkg/options/timeout.go

```diff
@@ -1,62 +1,34 @@
 package options
 
 import (
-	"github.com/Lilypad-Tech/lilypad/v2/pkg/data"
+	"github.com/Lilypad-Tech/lilypad/v2/pkg/solver"
 	"github.com/spf13/cobra"
 )
 
-func GetDefaultTimeoutOptions() data.DealTimeouts {
-	return data.DealTimeouts{
-		Agree: data.DealTimeout{
-			Timeout:    GetDefaultServeOptionUint64("TIMEOUT_AGREE_TIME", 3600),
-			Collateral: GetDefaultServeOptionUint64("TIMEOUT_AGREE_COLLATERAL", 1),
-		},
-		SubmitResults: data.DealTimeout{
-			Timeout:    GetDefaultServeOptionUint64("TIMEOUT_SUBMIT_RESULTS_COLLATERAL", 3600),
-			Collateral: GetDefaultServeOptionUint64("TIMEOUT_SUBMIT_RESULTS_COLLATERAL", 1),
-		},
-		JudgeResults: data.DealTimeout{
-			Timeout:    GetDefaultServeOptionUint64("TIMEOUT_JUDGE_RESULTS_COLLATERAL", 3600),
-			Collateral: GetDefaultServeOptionUint64("TIMEOUT_JUDGE_RESULTS_COLLATERAL", 1),
-		},
-		MediateResults: data.DealTimeout{
-			Timeout:    GetDefaultServeOptionUint64("TIMEOUT_MEDIATE_RESULTS_COLLATERAL", 3600),
-			Collateral: GetDefaultServeOptionUint64("TIMEOUT_MEDIATE_RESULTS_COLLATERAL", 1),
-		},
+func GetDefaultTimeoutOptions() solver.SolverTimeoutOptions {
+	return solver.SolverTimeoutOptions{
+		MatchSeconds:     GetDefaultServeOptionUint64("TIMEOUT_MATCH_SECONDS", 30),
+		ExecutionSeconds: GetDefaultServeOptionUint64("TIMEOUT_EXECUTION_SECONDS", 900), // 15 minutes
+		DownloadSeconds:  GetDefaultServeOptionUint64("TIMEOUT_DOWNLOAD_SECONDS", 3600), // One hour
+		TotalSeconds:     GetDefaultServeOptionUint64("TIMEOUT_TOTAL_SECONDS", 86400),   // One day
```

Comment:
Maybe we want to change the TotalSecond default to just be the sum of the default match, execution and download. One day seems like a lot 😅